onnxruntime/core/framework/session_state_flatbuffers_utils.cc

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/framework/session_state_flatbuffers_utils.h"
+
+namespace onnxruntime::experimental::utils {
+
+std::string GetSubgraphId(const NodeIndex node_idx, const std::string& attr_name) {
+  return std::to_string(node_idx) + "_" + attr_name;
+}
+
+FbsSessionStateViewer::FbsSessionStateViewer(const fbs::SessionState& fbs_session_state)
+    : fbs_session_state_{fbs_session_state} {
+}
+
+Status FbsSessionStateViewer::Validate() const {
```

Comment:
For the validate(), can we log the error message and return a boolean instead of status? Not sure if this will trim down any binary size

Reply:
FYI, tried it and observed a slight increase in SizeBench